submissions/NadiaVorontsova/a-tiny-JS-world-OOP/index.js

```diff
@@ -0,0 +1,91 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   
+   Code repository: https://github.com/NadiaVorontsova/a-tiny-JS-world-OOP
+   Web app: https://nadiavorontsova.github.io/a-tiny-JS-world-OOP/
+   */
+
+// ======== CLASS DEFINITIONS ========
+class Inhabitant {
+   constructor(species, name, gender, saying, friend) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+      this.friend = friend || 'no friend:(';
+   }
+   printInhabitant() {
+      return `I am a <strong>${this.species}</strong>; My name is <strong>${this.name}</strong>; I am a <strong>${this.gender}</strong>;
+      I say: <strong>'${this.saying}'</strong>; My friends: <strong>${this.friend}</strong>`;
```

Comment:
Not required to implement here. Just an idea for one of possible ways to add readability for long strings.

```
[
  "text",
  this.property,
  "more text",
  this.anotherProperty,
].join("; ");
```

Reply:
Thanks for this example. I used it)